common/brave_features.cc

```diff
@@ -19,4 +19,7 @@ const base::Feature kBraveRewards{"BraveRewards",
 #endif
 #endif  // defined(OS_ANDROID)
 
+const base::Feature kGlobalPrivacyControl{"GlobalPrivacyControl",
```

Comment:
can we add a comment, link to the spec?

Reply:
Spec is currently not public, so there isn't a stable URL for it at the moment unfortunately

Reply:
Perhaps just a comment or link to a wiki on brave-browser

Reply:
I added a comment from the spec